allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureDescriptionTest.java

```diff
@@ -0,0 +1,27 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Description;
+import org.junit.jupiter.api.Test;
+
+public class AllureDescriptionTest {
+
+    @Test
+    @Description("Static description")
```

Comment:
I often encountered that rendering multiline descriptions might be non-obvious... 
Can we do this in java's allure? how? should we extend this example correspondingly? 

If this is possible to make multiline description in some way using the @Description annotation – it would be good to provide such example here

Reply:
Here is a python version:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/1961118/112281170-6ef92b00-8c8e-11eb-8a36-ead403afce91.png">
Probably it would be good to see corresponding alternatives to multiline and html versions in java too...